packages/quantic/force-app/main/default/lwc/quanticBreadcrumbManager/quanticBreadcrumbManager.html

```diff
@@ -1,78 +1,78 @@
-                                        <a onclick={breadcrumb.expandButtonClick} class="slds-text-color_default breadcrumb-manager__more-button">{breadcrumb.showMoreButtonText}</a>
-                                    </li>
-                                </template>
-                            </ul>
-                        </li>
-                    </template>
-                </ul>
-            </div>
-            <div class="slds-col breadcrumb-manager__column">
-                <ul class="slds-list_horizontal slds-has-block-links_space">
-                    <li class="breadcrumb-manager__value-line-item">
-                        <a onclick={deselectAll}>{labels.clearAllFilters}</a>
-                    </li>
-                </ul>
-            </div>
-        </div>
-    </template>
+  <template if:true={hasBreadcrumbs}>
+    <div class="slds-grid slds-grid_vertical-align-start slds-size_1-of-1 slds-var-m-vertical_x-small">
+      <div class="slds-col slds-truncate slds-col_bump-right breadcrumb-manager__column">
+        <ul>
+          <template for:each={facetBreadcrumbValues} for:item="breadcrumb">
+            <li key={breadcrumb.facetId} class="slds-list_horizontal breadcrumb-manager__line">
+              <span class="slds-col slds-gutters breadcrumb-manager__field-name">{breadcrumb.field} : </span>
+              <ul class="slds-col slds-truncate slds-gutters slds-list_horizontal slds-grid slds-wrap">
+                <template for:each={breadcrumb.values} for:item="breadcrumbValue">
+                  <li key={breadcrumbValue.value.value} class="breadcrumb__container">
+                    <c-quantic-pill label={breadcrumbValue.value.value} alt-text={breadcrumbValue.value.value} deselect={breadcrumbValue.deselect}></c-quantic-pill>
+                  </li>
+                </template>
+                <template if:true={breadcrumb.showMoreButton}>
+                  <li>
+                    <a onclick={breadcrumb.expandButtonClick} class="slds-p-horizontal_x-small slds-p-vertical_xx-small slds-text-color_default breadcrumb-manager__more-button">{breadcrumb.showMoreButtonText}</a>
+                  </li>
+                </template>
+              </ul>
+            </li>
+          </template>
+          <template for:each={numericFacetBreadcrumbsValues} for:item="breadcrumb">
+            <li key={breadcrumb.facetId} class="slds-list_horizontal breadcrumb-manager__line">
+              <span class="slds-col slds-gutters breadcrumb-manager__field-name">{breadcrumb.field} : </span>
+              <ul class="slds-col slds-truncate slds-gutters slds-list_horizontal slds-grid slds-wrap">
+                <template for:each={breadcrumb.values} for:item="breadcrumbValue">
+                  <li key={breadcrumbValue.value.value} class="breadcrumb__container">
+                    <c-quantic-pill label={breadcrumbValue.value} alt-text={breadcrumbValue.value} deselect={breadcrumbValue.deselect}></c-quantic-pill>
```

Comment:
Why are you using `breadcrumbValue.value` instead of `breadcrumbValue.value.value` like you do with regular facet values?

I feel the terminology around the breadcrumb levels is not that clear.

Reply:
Yeah it could use some refactoring. I tried not to touch the javascript on this one so this is the way it was before this update but i can give it a look.

Reply:
I looked into it and that's how the type is defined in headless. The breadcrumb.value has a different type based on the facet filter variant. Instead of adding logic in the component to map the breadcrumb values to some other format depending on the type I would prefer to take and display the values returned from headless as-is.